File: Typescript/Practice/easy/easy_findPotatoes.ts
#!/usr/bin/env npx ts-node

function potatoes (letters: string): number {
  let counter: number = 0;
  for (let i = 0; i <= letters.length - 6; i++) {
    if (letters.slice(i, i+6) === 'potato') {
      counter++
    };
  };
  return counter;
};


console.log(potatoes("potato"));
console.log(potatoes("potatopotato"));
console.log(potatoes("potatoapple"));

File: Typescript/Practice/26072022/easy/easy_findPotatoes.py
#!/usr/bin/env python

def potatoes(string):
  counter = 0
  for i in range(len(string) - 5):
    if string[i:i+6] == 'potato':
      counter += 1
  return counter

print(potatoes("potato"))
print(potatoes("potatopotato"))
print(potatoes("potatoapple"))
